keylime/db/verifier_db.py

```diff
@@ -24,7 +25,8 @@ class VerfierMain(Base):
     public_key = Column(String(500))
     tpm_policy = Column(JSONPickleType(pickler=JSONPickler))
     meta_data = Column(String(200))
-    allowlist = Column(Text().with_variant(Text(429400000), "mysql"))
+    ima_policy = relationship("VerifierAllowlist", back_populates="agent", uselist=False)
```

Comment:
Let's take this opportunity to get the naming fixed/right. It's super weird to call something `ima_policy` in one place and `allowlist` in another. If the table is named `allowlist` then the foreign key should be as well.

So does it make more sense to have the table be `ima_policy` and the attribute be the `allowlist` or vice versa?

Reply:
I think it might make the most sense to postpone any comprehensive work around making naming consistent for a different pull request. This one is pretty hefty as is.